src/descriptors/power_system_structs.json

```diff
@@ -65,6 +65,69 @@
       ],
       "supertype": "OperationalCost"
     },
+    {
+        "struct_name": "StorageManagementCost",
```

Comment:
Not sure if StorageManagementCost is the best descriptor here.

Reply:
@claytonpbarrows what do you think about this name?

Reply:
I don't hate `StorageManagementCost`. But what about `StorageOperationalCost`?